pom.xml

```diff
@@ -286,11 +286,11 @@ THE SOFTWARE.
           </dependencies>
           <executions>
             <execution>
-              <id>validate</id>
```

Comment:
actually its check and all other executions reply the goal as well:


<img width="681" height="338" alt="Image" src="https://github.com/user-attachments/assets/0ae6da31-ccbe-462f-b21c-ee4a1888fcc5" />